
bool STOP_STATES
bool STOP_OPERATIONAL
bool STOP_MANUAL
bool STOP_PROTECTIVE
bool STOP_EMERGENCY
bool STOP_RESUME
